tests/conftest.py

```
@@ -0,0 +1,379 @@
+    # Grant geolocation permissions
+    context.grant_permissions(["geolocation"])
+
+    def set_location(latitude: float, longitude: float) -> None:
+        """Set the browser geolocation"""
+        context.set_geolocation({"latitude": latitude, "longitude": longitude})
+
+    return set_location
+
+
+@pytest.fixture
+def performance_tracker(page: Page) -> Callable:
```

Ruff (0.14.11)

Unused function argument: `page`

(ARG001)